packages/init/src/lib/auth.ts

```diff
@@ -0,0 +1,98 @@
+import { existsSync, readFileSync } from "node:fs";
```

Comment:
We're just putting auth-related code in a separate file. No changes were made to the authentication logic.